server/tserver/src/main/java/org/apache/accumulo/tserver/session/Session.java

```diff
@@ -66,13 +67,21 @@ public State getState() {
     return state;
   }
 
+  public void setSessionId(long sessionId) {
+    this.sessionId = sessionId;
+  }
+
+  public long getSessionId() {
+    return sessionId;
+  }
+
```

Comment:
Could make the getter and setters more strict, only allowing it to be set once and failing get when not set.


```suggestion
  private boolean sessionIdSet = false;

  public void setSessionId(long sessionId) {
    Preconditions.checkState(!sessionIdSet);
    this.sessionId = sessionId;
     sessionIdSet = true;
  }

  public long getSessionId() {
    Preconditions.checkState(sessionIdSet);
    return sessionId;
  }

```

Reply:
Done in bd186ca8f80fbc0de2554b1027880276f131ac8c